frontend/src/app/app.module.ts

```diff
@@ -11,35 +11,35 @@ import { CommonModule } from '@angular/common';
 import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
 import { NgModule } from '@angular/core';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
 import { MatBadgeModule } from '@angular/material/badge';
```

Comment:
URL: http://localhost:4200/project/coffee-machine/metadata
Issue: 
- Remove project configuration title, it's already clear from the breadcrumb

Screenshot:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/7cff7935-5d4d-439e-9804-0e89a52e93eb)